README.md

```diff
@@ -56,6 +56,12 @@ If you use the Habitat platform in your research, please cite the following [pap
   booktitle =     {Proceedings of the IEEE/CVF International Conference on Computer Vision (ICCV)},
   year      =     {2019}
 }
+@article{szot2021habitat,
```

Comment:
I personally like the newest paper first style. Thoughts @dhruvbatra @mathfac ?

Reply:
That makes sense to me.

Reply:
Sounds good to me too.